src/patternfly/components/Button/button.scss

```diff
@@ -271,7 +274,15 @@
   user-select: none;
   background-color: var(--#{$button}--BackgroundColor);
   border: 0;
-  border-radius: var(--#{$button}--BorderRadius);
+  border-start-start-radius: var(--#{$button}--start-start--Radius--BorderRadius);
+  border-start-end-radius: var(--#{$button}--start-end--Radius--BorderRadius);
+  border-end-start-radius: var(--#{$button}--end-start--Radius--BorderRadius);
+  border-end-end-radius: var(--#{$button}--end-end--Radius--BorderRadius);
+
+  // apply line-height to all children, including fa fonts
+  > * {
+    line-height: var(--#{$button}--LineHeight);
+  }
```

Comment:
Curious what this is needed for? Looks like the horizontal nav arrow icons align OK without it.